### PR TITLE
core: using math.MaxUint64 instead of 0xffffffffffffffff

### PR DESCRIPTION
**Pull request by cuiweixie**

### CODE REVIEW

File: core/vm/instructions.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/params"
 	"github.com/holiman/uint256"
+	"math"
```
Comment (rjl493456442) [inline core/vm/instructions.go:25]: Please fix the imports.

Comment (cuiweixie) [reply core/vm/instructions.go]: done

**Review by holiman — COMMENTED** [review]
Why though?

**Comment by cuiweixie** [comment]
> Why though? 

when I am reading the code, i have count f for 16times to make sure i am right about that It does is maxuint64. so I send this pr to make it more readable for reading the code.

**Comment by rjl493456442** [comment]
I think the change is good, `math.MaxUint64` is definitely more readable than `0xffffffffffffffff`

**Review by holiman — APPROVED** [review]
LGTM

**Pull request merged**